llvm/test/Transforms/VectorCombine/foldShuffleOfIntrinsics.ll

```diff
@@ -0,0 +1,81 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -passes=vector-combine -S %s | FileCheck %s
```

Comment:
You're running tests for a fold that uses a cost function comparison without a triple / cpu target?

Reply:
Is this a bad idea? Different CPUs may have different cost model, but the optimization should be good.

Reply:
It will mean it will take the cost model of whatever the native machine is (ppc, arm, x86, etc etc), which could give different results on different machines.